session/src/test/java/org/apache/iotdb/session/SessionUT.java

```diff
@@ -170,7 +170,7 @@ public void testInsertByStrAndSelectFailedData()
       RowRecord record = dataSet.next();
       int nullCount = 0;
       for (int j = 0; j < 4; ++j) {
-        if (record.getFields().get(j) == null) {
+        if (record.getFields().get(j) == null || record.getFields().get(j).getDataType() == null) {
```

Comment:
Why we need the statement `record.getFields().get(j).getDataType() == null` here?

Reply:
I added it because this UT failed on my local machine if missing it. Let me try to remove this and see the GitHub actions test results.

Reply:
All right, the tests passed. It seems this statement is not necessary....